static/app/views/preprod/header/buildDetailsHeaderContent.tsx

```diff
@@ -0,0 +1,133 @@
+import styled from '@emotion/styled';
+
+import {addErrorMessage} from 'sentry/actionCreators/indicator';
+import {Breadcrumbs, type Crumb} from 'sentry/components/breadcrumbs';
+import {Alert} from 'sentry/components/core/alert';
+import {Button} from 'sentry/components/core/button';
+import Placeholder from 'sentry/components/placeholder';
+import {IconEllipsis, IconTelescope} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {BuildDetailsApiResponse} from 'sentry/views/preprod/types';
+
+type BuildDetailsHeaderError = {error: string; status: 'error'};
+type BuildDetailsHeaderLoading = {status: 'loading'};
+type BuildDetailsHeaderSuccess = {
+  buildDetails: BuildDetailsApiResponse;
+  status: 'success';
+};
+
+export type BuildDetailsHeaderContentProps =
+  | BuildDetailsHeaderError
+  | BuildDetailsHeaderLoading
+  | BuildDetailsHeaderSuccess;
+
+export function BuildDetailsHeaderContent(props: BuildDetailsHeaderContentProps) {
+  const {status} = props;
+
+  if (status === 'loading') {
+    return (
+      <HeaderContainer>
+        <Placeholder height="20px" width="200px" style={{marginBottom: space(2)}} />
```

Comment:
Skeleton loaders, a bit tough to see with the current header coloring

<img width="1489" height="118" alt="Image" src="https://github.com/user-attachments/assets/19628c38-6f31-4909-a094-011c630e7190" />